service/src/main/java/uk/gov/hmcts/reform/fpl/exceptions/GrantCaseAccessException.java

```diff
@@ -21,7 +21,7 @@ public GrantCaseAccessException(String caseId, Set<String> userIds, Set<CaseRole
     }
 
     public GrantCaseAccessException(String caseId, String localAuthority, Set<CaseRole> caseRoles, Exception e) {
-        super(String.format("User from %s not granted %s to case %s", localAuthority, caseRoles, caseId), e);
+        super(String.format("Users from %s not granted %s to case %s", localAuthority, caseRoles, caseId), e);
```

Comment:
The above has it as `User(s)`, we should probably be consistent and go with your change for both